src/WebSocketMiddleware.php

```diff
@@ -12,14 +12,23 @@
 final class WebSocketMiddleware
 {
     private $paths;
-    private $connectionHandler = null;
+    private $connectionHandler;
     private $subProtocols;
+    private $negotiator;
 
-    public function __construct(array $paths = [], callable $connectionHandler = null, array $subProtocols = [])
+    public function __construct(array $paths = [], callable $connectionHandler = null, array $subProtocols = [], $negotiator = null)
```

Comment:
This should be typed `NegotiatorInterface`

Reply:
@mbonneau Added.